pkgs/by-name/aw/aws-lc/package.nix

```diff
@@ -0,0 +1,92 @@
+{
+  lib,
+  stdenv,
+  overrideSDK,
+  cmakeMinimal,
+  fetchFromGitHub,
+  ninja,
+  testers,
+  aws-lc,
+  useSharedLibraries ? !stdenv.targetPlatform.isStatic,
+  ...
```

Comment:
```suggestion
```

Reply:
I only had this because I was trying to override openssl in another flake with aws-lc and it was causing issues without it. I think it's a good idea to remove the `...` now since it should be fixed in the stdenv bootstrapping side of things 👍🏼